src/components/SchemaView/SchemaViewMain.vue

```diff
@@ -242,18 +242,21 @@ export default {
       return relTable.group ? relTable.group : relTableName;
     },
     getLayoutConfig(edges) {
-      const nodeSpacing = edges.length * 5;
       const config = {
-        type: 'comboForce',
+        type: 'force',
         preventOverlap: true,
         preventNodeOverlap: true,
         preventComboOverlap: true,
-        linkDistance: 250,
-        nodeStrength: 100,
+        linkDistance: 30,
+        nodeStrength: 0.1,
         nodeSize: 100,
-        nodeSpacing,
         comboSpacing: 10,
         comboCollideStrength: 0.2,
+        edgeStrength: 0.1,
```

Comment:
After changing back to `comboForce`, please verify if these configuration is still valid.